core/src/main/scala/org/broadinstitute/workbench/hamm/Main.scala

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.workbench.hamm
+
+import cats.effect._
+import cats.implicits._
+import fs2._
+import org.broadinstitute.workbench.hamm.api.HammRoutes
+import org.broadinstitute.workbench.hamm.auth.HttpSamDAO
+import org.broadinstitute.workbench.hamm.dao.{GooglePriceListDAO, WorkflowMetadataDAO}
+import org.broadinstitute.workbench.hamm.service.{StatusService, WorkflowCostService}
+import org.http4s.Uri
+import org.http4s.client.blaze.BlazeClientBuilder
+import org.http4s.server.blaze.BlazeServerBuilder
+
+import scala.concurrent.ExecutionContext.Implicits.global //use better thread pool
+
+object Main extends IOApp with HammLogger {
```

Comment:
this should live in `server` package...we have a background service depends on `core` as well, and we definitely want to run api server tgt with the background process